### ISSUE TITLE
Investigate appropriate network for face detection

### ISSUE THREAD

**Issue by SnirX** [SnirX/personal-face-detector#1]
For example:
Yolov3
OpenCV

**Comment by marielcherkassky** [comment]
OpenCV - Haar Cascades Classifier
Dlib - Histogram of Oriented Gradients (HOG) + SVM
DeepFace
FaceNet - implemented in pytorch http://cmusatyalab.github.io/openface/
- implemented in keras - https://machinelearningmastery.com/how-to-develop-a-face-recognition-system-using-facenet-in-keras-and-an-svm-classifier/

OPENCV vs YOLO real time camera - https://www.youtube.com/watch?v=XkfSqOvJRIw

**Comment by marielcherkassky** [comment]
bottom line : yolo generates better results, by far faster in real time.
